dir.traversal.targets

```diff
@@ -86,6 +86,32 @@
              ContinueOnError="ErrorAndContinue" />
   </Target>
 
+  <Target Name="ProducesPackageId"
+          Returns="@(PackageIds)" 
```

Comment:
cc @ericstj 

Do we want to gather this by calling this target on all our projects? Shouldn't this be scoped to just pkgproj's? Also is it worth just getting the list from the produced packages in the output?